### ISSUE TITLE
Convert between Windows and POSIX paths

### ISSUE THREAD

**Issue by bdbenim** [bdbenim/stash-empornium#145]
Currently all paths are assumed to be POSIX-style, which does not work for Windows users running Stash natively.

**Comment by Smoukus** [comment]
I think this is a major hurdle for Windows users.

**Comment by bdbenim** [comment]
Yes, unfortunately since I don't run Stash on Windows it's not easy for me to do a lot of development in this area. If someone is willing to submit a PR it would be appreciated. Or at the very least some examples of paths that aren't working with the app so that I can write unit tests
